README.md

```diff
@@ -10,6 +10,7 @@ It also includes a shell utility to pretty-print its log files.
 * [Benchmarks](#benchmarks)
 * [API](#api)
 * [How do I rotate log files?](#rotate)
+* [What happened to my transports?](#transports)
```

Comment:
Hmmm, I'm not sure, transports aren't _in_ Pino, they're outside of Pino, so maybe `How to use Transports with Pino`